sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -688,6 +688,13 @@ object RapidsConf {
     .stringConf
     .createWithDefault("NONE")
 
+  val SHIMS_PROVIDER_OVERRIDE = conf("spark.rapids.shims-provider-override")
+    .internal()
+    .doc("Overrides the automatic Spark shim detection logic and forces a specific shims " +
+      "provider class to be used. Set to the fully qualified shims provider class to use.")
```

Comment:
I wonder if it would be helpful to add a sentence that says if you have a custom spark version with custom version such as 3.0.0.0, this can be used to set to the shim version that would match the base Spark version, ie com.nvidia.spark.rapids.shims.spark300.SparkShimServiceProvider. If you modified Spark there is no guarantee this will work.

Reply:
I updated the doc string.